specs/plugin-extensibility.md

```diff
@@ -22,13 +22,28 @@ Keys and associated certificates used for signing artifacts using Notation could
 
 Notation will invoke plugins as executable, pass parameters using command line arguments, and use standard IO streams to pass request/response payloads. This mechanism is used as Go language (used to develop [Notation Go library](https://github.com/notaryproject/notation-go)) does not have a [in built support](https://github.com/golang/go/issues/19282) to load and execute plugins that works across OS platforms. Other mechanisms like gRPC require every plugin to be implemented as a service/daemon.
 
+### Package and distribute plugin file
```

Comment:
This is a subtitle of `plugin mechanism`
```suggestion
### Package and release
```

Reply:
This paragraph is not for the plugin release process, it's only for how to correctly package a plugin. Release is much more complex